e2e-tests/cypress/integration/appeal-submission-appeal-site-address.feature

```diff
@@ -0,0 +1,93 @@
+@wip
+Feature: Appellant provides the Appeal Site Address
+    I need to provide the appeal site address, so that all parties know which site the appeal is against.
+
+    Scenario Outline: Prospective Appellant submits a valid appeal site address
+        Given the user is prompted for the site address
+        When the user provides their appeal site address as <Address Line 1> and <Address Line 2> and <Town or City> and <County> and <Postcode>
+        Then the user is able to continue with the provided address
+        #And the user can see that their appeal has been updated with the provided address
```

Comment:
Are these intended to be comments intended to be?

Reply:
Hi Frank, 
Yes, Ian was happy to leave them as comments for now and if needed that steps can be implemented in future.